### ISSUE TITLE
How to set different resolutions for training?

### ISSUE THREAD

**Issue by tms2003** [roboflow/rf-detr#53]
How to set different resolutions for training? How many epochs can generally achieve good results with fine-tuning? I saw that your sample code is 10 epochs, is it enough?

**Comment by isaacrob-roboflow** [comment]
You can change the resolution for training by setting resolution at model instantiation, like so:

```
from rfdetr import RFDETRBase

model = RFDETRBase(resolution=448)

model.train(dataset_dir=<DATASET_PATH>, epochs=10, batch_size=4, grad_accum_steps=4, lr=1e-4)

```

Our RF100VL results used 100 epochs for all 100 datasets. The best value depends on your specific dataset. In general I'd say err towards using more and just take the version based on validation loss.

**Comment by SkalskiP** [comment]
I’m closing this issue for now. I added the relevant information to README.md. @tms2003 feel free to reopen the issue if you’ll have more questions.
